crates/ruff_workspace/src/configuration.rs

```diff
@@ -660,7 +660,12 @@ impl LintConfiguration {
             .collect();
 
         #[allow(deprecated)]
-        let ignore_init_module_imports = options.common.ignore_init_module_imports;
+        let ignore_init_module_imports = {
+            if options.common.ignore_init_module_imports.is_some_and(|t| t) {
```

Comment:
I think we should warn regardless of the value if we're going to remove the setting, right? I guess it depends if preview is enabled but is that feasible to check here? i.e. if preview is disabled and this is false the suggestion is to try preview mode instead but not everyone can do that since it's a global toggle.

Reply:
Oops! Yes :100: thank you for catching this.